### ISSUE TITLE
Can't resolve Chucker via Jitpack

### ISSUE THREAD

**Issue by bhavika-traderev** [ChuckerTeam/chucker#877]
## :writing_hand: Describe the bug
I ma getting following error while executing CI :
<img width="881" alt="Screen Shot 2022-09-08 at 3 37 09 PM" src="https://user-images.githubusercontent.com/104368901/189230589-8aaa95fa-5871-413a-9f41-d5f9b7b80faf.png">

## :iphone: Tech info
 - Device: 
 - OS: 
 - Chucker version:

**Comment by vbuberen** [comment]
Jitpack has issues with availability from time to time for a long time already. This is why we started to distribute Chucker via Maven Central.

If you have no other libraries resolved via Jitpack just drop it and be sure that you get Chucker via Maven Central.

**Comment by vbuberen** [comment]
Yes, it is added correctly, but your `repositories` section probably has something like this:

```
repositories {
    maven { url "https://jitpack.io" }
    mavenCentral
    ...
}
```
Due to that during the build library is resolved via Jitpack, not Maven Central. I would highly recommend to drop Jitpack completely as most libraries already migrated from it or available both on Jitpack and Maven Central like Chucker.

**Comment by cortinico** [comment]
As @vbuberen suggested, consider switching to Maven Central.

I also noticed that the build log for 3.5.2 is missing on Jitpack.
https://jitpack.io/com/github/ChuckerTeam/Chucker/3.5.2/build.log

I've triggered a new job and artifacts should be available in minutes.

**Comment by bhavika-traderev** [comment]
Some libraries in my project depends on jitpick. So I cant remove it.

**Comment by cortinico** [comment]
> Some libraries in my project depends on jitpack. So I cant remove it.

You can restrict which libraries are pulled from which repositories with:

```
repositories {
  maven {
    url "https://jitpack.io"
    content {
      // Exclude Chucker from Jitpack
      includeGroupByRegex "com\\.github\\.ChuckerTeam"
    }
  }
}
```

Docs on this are here: https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:declaring-repository-filter

Closing as this issue is solved (3.5.2 is back on Jitpack als)
